CMakeLists.txt

```diff
@@ -324,6 +324,7 @@ if (WITH_SYSTEM_PROVIDED_3PARTY)
   find_package(gflags REQUIRED)
 
   find_package(expat CONFIG REQUIRED)
+  find_package(jansson CONFIG REQUIRED)
```

Comment:
What is wrong with building these packages from sources?..

Reply:
Nothing in particular, though mixing system and vendored deps is considered bad.
Note that `add_subdirectory(jansson)` is not called on when `WITH_SYSTEM_PROVIDED_3PARTY`. CMake implicitly uses `-ljansson` if it couldn't find target with name `jansson`. It happens to work on Linux because everything is in include/link directories. This is why namespaced targets are preferred (they don't allow this implicit behavior).